chord_metadata_service/patients/migrations/0004_v2_19_0.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 4.1.7 on 2023-03-10 18:50
```

Comment:
this is called v2_19_0, but really this should be a major release due to the breaking changes - v3.0.0